canonical-web-and-design/maas-ui
# build(all): upgrade to Vanilla 2.31.0

## Done

- Upgrade to Vanilla 2.31.0.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Open the machine list and check that the tabs don't cover the border above it.
- Load the controller details and check that the tabs don't cover the border above it.
- Have a look around and make sure nothing else is broken.
